aries-site/src/pages/components/button.mdx

```diff
@@ -49,6 +49,35 @@ The HTML elements for buttons and links (a.k.a. anchors) describe a very specifi
 - Use buttons when you are performing an action, such as: "submit," "merge," "create new," "upload," etc.
 - An action almost always occurs on the same page
 
+### Toggle Buttons
+
+Toggle buttons are buttons with either two states: check and unchecked or used to trigger two actions like on or off.
+
+#### Toggle Buttons with Icons
+
+Icon buttons have an aria-label that will describe the action that the user will be performing when using screen readers.
+
+**Best Practices:**
+- Don’t use more than 2-3 words
+- Use verbs first to describe the action such as “Toggle” , “Favorite” or “Save”
+- Use sentece case capitalization
+- It is recommended that words such as “the”, “a” be avoided
+
+For Buttons with Icons ONLY, it is important that you show the two following elements:
+
+1. Show the current state of the button through the use of an icon that best represents the action
```

Comment:
```suggestion
1. Show the current state of the button through the use of an icon that best represents the action.
```

Reply:
It would be helpful to supplement this with an example such as show/hide password, or lock / unlock editing mode, or even the play/pause.

Reply:
I agree. I've added a ticket in our backlog to add this example along with some guidance. Ticket #1840